pycbc/events/stat.py

```diff
@@ -28,33 +28,6 @@
 import numpy
 from . import events
 
-def get_statistic(option, files):
```

Comment:
OK, so the point of defining a function would be to ensure a more user-friendly error message in the possible future case of some other script wanting to use the same classes? 
At the moment there *is* nothing else to be compatible with.  In any case function calls should be able to change between code versions.